### ISSUE TITLE
URL in address bar does not change after publishing story

### ISSUE THREAD

**Issue by swissspidy** [GoogleForCreators/web-stories-wp#697]
## Bug Description

`refreshPostEditURL` is used to update the URL in the address bar after publishing/updating a story.

However, it currently only runs after updating a story, not after initially publishing it.

## Expected Behaviour

URL in address bar should be updated the first time around, when publishing the story.

## Steps to Reproduce

1. Create new story.
1. Click `Publish`
1. Notice that URL does not change, it stays `wp-admin/post-new.php?post_type=web-story`
1. Click `Update
1. Notice that URL changes to `wp-admin/post.php?post=123456&action=edit`

In other words:

1. Create new story.
1. Click `Publish`
1. Reload page
1. Lose changes

## Screenshots

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._
